test/api/v01/request_helper.rb

```diff
@@ -118,6 +119,7 @@ def asynchronously(options = {start_worker: false})
     OptimizerWrapper.config[:solve][:synchronously] = false
     old_resque_inline = Resque.inline
     Resque.inline = false
+    sleep 0.5
```

Comment:
It has been removed once by @senhalil . Do you remember why ?

Reply:
because it's not necessary. It was also me (I believe) who added it in the first place.

Reply:
Back then we had race condition in the public API, so it was a way to avoid them if my memory serves well.